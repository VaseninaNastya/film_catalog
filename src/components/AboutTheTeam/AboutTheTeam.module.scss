@import "../../index.scss";



.aboutTheTeam_imgWrapper{
  max-height: calc(100vh - 100px);
  width: 50%;
  display: flex;
  align-items: center;
  img{
    height: 100%;
    margin: 0 auto;
  }
}
.aboutTheTeam_description{
  padding: 0 20px;
  padding-top: 30px;
  h2{
    margin-bottom: 30px;
  }
}