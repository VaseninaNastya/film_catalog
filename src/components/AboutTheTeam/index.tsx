import React from "react";
import s from "./AboutTheTeam.module.scss";
import img from '../../img/me.jpg';

const AboutTheTeam:React.FunctionComponent = () => {
  return(
    <div className={s.container}>
      <div className={s.aboutTheTeam_description}>
        <h2>Vasenina Nastya</h2>
        <p>I worked as a civil engineer from 2012 to 2020. <br/>Then, from March 2021 to August 2021, I worked as an HTML coder. <br/>From 2019 to 2021 I have been studying JavaScript. <br/>In my free time I like to program, run, meet friends, go to museums and ride a bike.</p>
      </div>
      <div className={s.aboutTheTeam_imgWrapper}>
        <img src={img} alt="me"/>
      </div>
    </div>
  )
}

export default AboutTheTeam;
