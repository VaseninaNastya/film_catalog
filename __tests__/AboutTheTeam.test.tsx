import React from 'react';
import AboutTheTeam from '../src/components/AboutTheTeam'
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
  describe('src/components/AboutTheTeam', () => {
    describe('AboutTheTeam', () => {
      it('Capturing Snapshot of AboutTheTeam', () => {
        Enzyme.configure({ adapter: new Adapter() })
        const renderedValue =  shallow(
          <AboutTheTeam />
        )
        expect(renderedValue).toMatchSnapshot();
    });

    })



});









