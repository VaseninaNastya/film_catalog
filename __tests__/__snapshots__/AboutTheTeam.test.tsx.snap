// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/AboutTheTeam AboutTheTeam Capturing Snapshot of AboutTheTeam 1`] = `
<div
  className="container"
>
  <div
    className="aboutTheTeam_description"
  >
    <h2>
      Vasenina Nastya
    </h2>
    <p>
      I worked as a civil engineer from 2012 to 2020. 
      <br />
      Then, from March 2021 to August 2021, I worked as an HTML coder. 
      <br />
      From 2019 to 2021 I have been studying JavaScript. 
      <br />
      In my free time I like to program, run, meet friends, go to museums and ride a bike.
    </p>
  </div>
  <div
    className="aboutTheTeam_imgWrapper"
  >
    <img
      alt="me"
      src={null}
    />
  </div>
</div>
`;
